students/Roy_Tate/lesson03/list_lab.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+import sys
+
+
+space = ' '
+ast = '*'
+
+
+# Series 1 #
+
+print("\n" + ast * 5 + " Series 1 " + ast * 5 + "\n")
+fruit_list = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(fruit_list)
+user_fruit = input('Type another fruit and add it to the list: ')
+fruit_list.append(user_fruit)
+print(fruit_list)
+user_index = int(input('Enter a number and the fruit will be returned: '))
+if 0 <= user_index < len(fruit_list):
+    print(fruit_list[user_index -1])
+else:
+    print("The number is out of range.")
+# Add another fruit to the beginning of the list using “+” and display the list.
```

Comment:
looks like you're missing addition using `+` -> `fruit_list = ['FirstFruit'] + fruit_list`